tox.ini

```diff
@@ -22,11 +22,16 @@ deps =
     dj18: django~=1.8.0
     dj19: django~=1.9.0
     dj110: django~=1.10.0
+    dj111: django~=1.11.0
     wt12: Wagtail~=1.2.0
     wt13: Wagtail~=1.3.0
     wt14: Wagtail~=1.4.0
     wt15: Wagtail~=1.5.0
     wt16: Wagtail~=1.6.0
+    wt17: Wagtail~=1.8.0
```

Comment:
These numbers are mismatching. Also see the previous comment. Its about adopting wt18, wt19, wt110, wt111.